src/timeline/Timeline.tsx

```diff
@@ -152,8 +158,8 @@ const Timeline = (props: TimelineProps) => {
     return map(pageDates, d => groupedEvents[d] || []);
   }, [pageDates, groupedEvents]);
   const scrollView = useRef<ScrollView>();
-  const calendarHeight = useRef((end - start) * HOUR_BLOCK_HEIGHT);
-  const styles = useRef(styleConstructor(theme || props.styles, calendarHeight.current));
+  const calendarHeight = useMemo(() => (end - start) * hourBlockHeight, [hourBlockHeight]);
+  const styles = useMemo(() => styleConstructor(theme || props.styles, calendarHeight), [calendarHeight]);
```

Comment:
There are some places in this file where you still use `styles.current...` even tough it's not a ref anymore